contracts/truefi2/LoanFactory2.sol

```diff
@@ -135,6 +156,18 @@ contract LoanFactory2 is ILoanFactory2, Initializable {
         emit LoanTokenCreated(newToken);
     }
 
+    function createDebtToken(
+        ITrueFiPool2 _pool,
+        address _borrower,
+        uint256 _debt
+    ) external onlyTCA {
+        address newToken = address(new DebtToken());
+        DebtToken(newToken).initialize(_pool, lender, _borrower, liquidator, _debt);
```

Comment:
I'm sure this will be refactored in the next PR when we switch to proxies. No need to cast to address and then back to DebtToken.